setup.py

```diff
@@ -113,6 +113,7 @@ def run(self):
     'bleach',
     'typing_extensions',
     'pandas >=1.2',
+    'watchfiles'
```

Comment:
Add a min pin maybe? This avoids solvers picking up an old version we never tested against.